pkgs/development/interpreters/lua-5/interpreter.nix

```diff
@@ -56,7 +59,12 @@ self = stdenv.mkDerivation rec {
     "PLAT=${plat}"
     "CC=${stdenv.cc.targetPrefix}cc"
     "RANLIB=${stdenv.cc.targetPrefix}ranlib"
-  ];
+  ] ++ (if staticOnly then [
+    # Lua links with readline wich depends on ncurses. For some reason when
+    # building pkgsStatic.lua it fails because symbols from ncurses are not
+    # found. Adding ncurses here fixes the problem.
+    "MYLIBS=-lncurses"
+  ] else []);
```

Comment:
is it possible to add it all time? This may prevent some regressions from going unnoticed

Reply:
This will require many rebuilds. I am trying to avoid that. Should I change it anyway?

Reply:
as you can see with the ofborg badge, there are rebuilds anyway.

Reply:
Ok, I made the change.